packages/map-popup/src/index.tsx

```diff
@@ -101,7 +101,7 @@ function entityIsStation(entity: Entity): entity is Station {
 /**
  * Renders a map popup for a stop, scooter, or shared bike
  */
-export function MapPopup({ closePopup = null, configCompanies, entity, getEntityName, setLocation, setViewedStop }: Props): JSX.Element {
+export function MapPopup({ closePopup = () => null, configCompanies, entity, getEntityName, setLocation, setViewedStop }: Props): JSX.Element {
```

Comment:
```suggestion
export function MapPopup({ closePopup = () => void, configCompanies, entity, getEntityName, setLocation, setViewedStop }: Props): JSX.Element {
```
Should this be void instead?

Reply:
This isn't a type it's a default!

Reply:
We have a noOp function you can import